src/Stratis.Features.FederatedPeg/InputConsolidation/ConsolidationTransaction.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NBitcoin;
+using Stratis.Features.FederatedPeg.Interfaces;
+
+namespace Stratis.Features.FederatedPeg.InputConsolidation
+{
+    public class ConsolidationTransaction
+    {
+        public Transaction PartialTransaction { get; set; }
+
+        /// <summary>
+        /// Note this should never be Suspended or Rejected. Only Partial, FullySigned or SeenInBlock
```

Comment:
Could you enforce this behavior in the setter so it throws if you try to set it to `Suspended` or `Rejected`?